front-dispatcher/src/features/DashboardPage/components/DropDownsContainer/DropDowns.tsx

```diff
@@ -27,7 +20,7 @@ const DropDowns = ({ searchMainQuery }: props) => {
         if (cat && apiStrings[cat]) {
```

Comment:
The use of cat is not clarified enough, use other name that describes your intention (like category)